### PR TITLE
Create scraper scheduler.

### PR DESCRIPTION
**Pull request by gannerr**
Create scraper scheduler.
- Only runs at 1AM everyday for low latency
- Runs once automatically at startup to ensure our database is at least stored with preliminary data
- Using cron job so as to not rely on having the admin page open 24/7

### CODE REVIEW

**Comment by gannerr** [comment]
Screenshot 2020-08-26 at 9 07 18 PM
Created cron job to scraped data everyday at 1am

**Comment by graceehuu** [comment]
LGTM! using cron was a good choice :)

File: capstone/src/main/java/com/google/sps/servlets/SmallOwnedRestaurantsDataServlet.java [hunk 1]
```diff
@@ -100,4 +106,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
     response.sendRedirect("/admin.html");
   }
+
+  public void clearDatastore(){
+    Query restaurantQuery = new Query("Restaurant");
```
Comment (graceehuu) [inline capstone/src/main/java/com/google/sps/servlets/SmallOwnedRestaurantsDataServlet.java:111]: We'll have to change the name once we integrate

Comment (gannerr) [reply capstone/src/main/java/com/google/sps/servlets/SmallOwnedRestaurantsDataServlet.java:120]: for sure

File: capstone/src/main/java/com/google/sps/servlets/BlackOwnedRestaurantsDataServlet.java [hunk 2]
```diff
@@ -60,40 +56,51 @@ public void init() {
...
       for (PlaceDetails.Review review : reviewsList) {
         reviews += review.text + " ";
-      } 
-      
-    //   Set<String> tags = details.getTags(reviews);
-    //   String tagsString = tags.toString();
+      }
+
+      //   Set<String> tags = details.getTags(reviews);
+      //   String tagsString = tags.toString();
```
Comment (gannerr) [inline capstone/src/main/java/com/google/sps/servlets/BlackOwnedRestaurantsDataServlet.java:76]: We commented out tags and tagsString out as we are exploring a more manual approach to matching tags vs. NLP API which is inconsistent, buggy, and slow. We don't delete just so it is easier to transition back to NLP API if we find that manual approach is slower.

**Pull request merged**